atoma-proxy/src/server/middleware.rs

```diff
@@ -1820,7 +1834,7 @@ pub mod auth {
             return Ok(SelectedNodeMetadata {
                 stack_small_id: stack.stack_small_id,
                 selected_node_id: stack.selected_node_id,
-                tx_digest: None,
+                tx_digest: stack.tx_digest,
```

Comment:
It seems to me that sometime we return base58 encoded tx digest, sometimes not..

Reply:
This is because upon first acquisition of a stack, the `tx_digest` is encoded as `base58` string and then stored in the db, subsequent retrievals such as this case, of that `tx_digest` would be from the db and therefore it would already be a `String` and would not be required to be re-encoded